docs/src/pages/[platform]/connected-components/account-settings/change-password/react.mdx

```diff
@@ -1,3 +1,359 @@
-## Change Password
+import { Alert, Expander, ExpanderItem } from '@aws-amplify/ui-react';
 
-TODO
+import ReactPropsTable from '@/components/propsTable/ReactPropsTable';
+import { Example, ExampleCode } from '@/components/Example';
+import { ERROR_MESSAGE } from '../props';
+import { CHANGE_PASSWORD, OVERRIDES, PASSWORD_FIELDS, SUBMIT_BUTTON } from './props';
+
+
+### Props
+
+<ReactPropsTable props={CHANGE_PASSWORD} />
+
+### Basic Usage
+
+`ChangePassword` has `onSuccess` handler that will be called after successful password change. You may use this callback to run any custom handling, such as navigating route or rendering confirmation messages.
+
+<Example>
+  <ExampleCode>
+
+```jsx
+import React from 'react';
+import { AccountSettings } from '@aws-amplify/ui-react';
+
+function App() {
+  const handleSuccess = () => {
+    alert('password is successfully changed!')
+  }
+  
+  return (
+    <AccountSettings.ChangePassword onSuccess={handleSuccess}/>
+  );
+}
+```
+
+  </ExampleCode>
+</Example>
+
+### Custom Validation
+
+You can override the default custom password validator with your own. To do so, you can use the validators prop, which takes an array of ValidatorOptions, which is of type:
```

Comment:
Suggestion:

"You can override the default password validator with your own custom validator. To do so, you can use the `validator` prop, which takes an array of `ValidatorOptions`:"

Reply:
```suggestion
You can override the default password validator with your own custom validator. To do so, you can use the `validators` prop, which takes an array of `ValidatorOptions`:
```